batch/batch/cloud/azure/driver/create_instance.py

```diff
@@ -163,25 +163,26 @@ def create_vm_config(
 
 INTERNAL_GATEWAY_IP=$(jq -r '.internal_ip' userdata)
 
-# private job network = 10.0.0.0/16
-# public job network = 10.1.0.0/16
+# private job network = 172.20.0.0/16
+# public job network = 172.21.0.0/16
```

Comment:
What's up with the 172 change? Was this because we were conflicting with something? Didn't we also use these IP address ranges in some tests in test_batch.py?

Reply:
We realized that GCP uses the whole [Class A](https://www.ibm.com/docs/en/networkmanager/4.2.0?topic=translation-private-address-ranges) private IP range for nodes on the network, not just 10.128.x.x which turns out to be just our main zone. So we instead chose to switch to Class B to ensure job IPs could not conflict with node IPs. When I spun up my own GCP project for the n'th time, the database got assigned an IP in the old range and jobs couldn't talk to it.

There is a test that uses a 10.128.x.x to ensure that a job cannot communicate with a different worker on the network, but since that's a worker IP not a job IP that shouldn't need to change.